src/tests/unit/api/api.test.ts

```diff
@@ -139,4 +139,53 @@ describe("IBApi Tests", () => {
 
     ib.reqPnLSingle(44, _account, null, _conId);
   });
+
+  it("Test request tick history", async function (done) {
+    const ib = new IBApi({
+      clientId: _clientId++,
+      host: TEST_SERVER_HOST,
+      port: TEST_SERVER_POST,
```

Comment:
Once the test env PR is merged

The only valid values are `IB_HOST` and `IB_PORT`